client/src/pages/Contest.js

```diff
@@ -18,18 +18,20 @@ import MuiAlert from "@material-ui/lab/Alert";
 import AttachMoneyOutlinedIcon from "@material-ui/icons/AttachMoneyOutlined";
 import CheckCircleOutlineIcon from "@material-ui/icons/CheckCircleOutline";
 import DateFnsUtils from "@date-io/date-fns";
-
+import { useAuth } from "../components/UserContext";
 import {
   MuiPickersUtilsProvider,
   KeyboardTimePicker,
   KeyboardDatePicker,
 } from "@material-ui/pickers";
-
+import { Redirect } from 'react-router-dom'
```

Comment:
I forgot to add router when I was creating create contest. Are we going to use Redirect for navigating back to profile or last page? I added Link but somehow its not working( <Link to='/profile'>Back to profile</Link>). Demo will be easier if we have navigation on this page